src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,43 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.Person;
+
+/**
+ * Represents the association between Person and Tag.
+ * Contains the history of the tag that linked to the person and the tag type(Added or Deleted).
```

Comment:
Based on the given class diagram, this wasn't an intended use of the class. If it was, the diagram sould have been like this: 
![image](https://cloud.githubusercontent.com/assets/1673303/18409597/69111e4e-777d-11e6-92bf-9914c825bd13.png)